test/DynamoCoreTests/Logging/AnalyticsServiceTest.cs

```diff
@@ -41,42 +38,29 @@ public void DisableAnalytics()
             {
                 DynamoShapeManager.Utilities.GetInstalledAsmVersion2(versions, ref locatedPath, coreDirectory);
             }
+
             try
             {
-                Assert.DoesNotThrow(() =>
-                {
+                dynamoCLI = Process.Start(new ProcessStartInfo(Path.Combine(coreDirectory, "DynamoCLI.exe"), $"--GeometryPath \"{locatedPath}\" -k --DisableAnalytics -o \"{openPath}\" ") { UseShellExecute = true });
 
-                    dynamoCLI = Process.Start(new ProcessStartInfo(Path.Combine(coreDirectory, "DynamoCLI.exe"), $"--GeometryPath \"{locatedPath}\" -k --DisableAnalytics -o \"{openPath}\" ") { UseShellExecute = true });
-
-                    Thread.Sleep(5000);// Wait 5 seconds to open the dyn
-                    Assert.IsFalse(dynamoCLI.HasExited);
-                    var dt = DataTarget.AttachToProcess(dynamoCLI.Id, false);
-                    var assemblies = dt
-                          .ClrVersions
-                          .Select(dtClrVersion => dtClrVersion.CreateRuntime())
-                          .SelectMany(runtime => runtime.AppDomains.SelectMany(runtimeAppDomain => runtimeAppDomain.Modules))
-                          .Select(clrModule => clrModule.AssemblyName)
-                          .Distinct()
-                          .Where(x => x != null)
-                          .ToList();
+                Thread.Sleep(5000);// Wait 5 seconds to open the dyn
+                Assert.IsFalse(dynamoCLI.HasExited);
 
-                    var firstASMmodulePath = string.Empty;
-                    foreach (string module in assemblies)
+                var firstASMmodulePath = string.Empty;
+                foreach (ProcessModule module in dynamoCLI.Modules)
+                {
+                    if (module.ModuleName.IndexOf("Analytics.dll", StringComparison.OrdinalIgnoreCase) != -1)
```

Comment:
Are all of these modules managed modules? Will this property return both managed and unmanaged?

Reply:
both, managed and unmanaged